code/datums/effects/bleeding.dm

```diff
@@ -85,6 +91,11 @@
 	effect_name = "internal bleeding"
 	flags = INF_DURATION | NO_PROCESS_ON_DEATH | DEL_ON_UNDEFIBBABLE
 
+/datum/effects/bleeding/internal/Destroy()
+	if(limb)
+		SEND_SIGNAL(limb, COMSIG_LIMB_STOP_BLEEDING, FALSE, TRUE)
```

Comment:
this should only be sent from a single place.
Don't send signals from multiple places within code.
Either send this from `/datum/effects/bleeding/Destroy()` or separate it out into its own signals